packages/IntegrationTest/tests/MountComponentTests.tsx

```diff
@@ -0,0 +1,199 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ */
+
+import * as React from 'react';
+import * as RN from 'react-native';
+import {useEffect, useRef} from 'react';
+import {assert} from 'chai';
+import {componentTest, TestComponentType} from './lib/TestDefinition';
+
+function onNativeRender(callback: () => void) {
+  // We need to wait until native has rendered a frame before measuring will
+  // return non-zero results. Use RAF to schedule work on the next render, to
+  // then shceduled work on the render after (at which point we should be all
+  // good).
+  requestAnimationFrame(() => {
+    requestAnimationFrame(() => {
+      callback();
+    });
+  });
+}
+
+function mountAndMeasure(
+  Component: React.ComponentType<any>,
+): TestComponentType {
+  return ({pass}) => {
+    const ref = useRef<React.Component<any> & Readonly<RN.NativeMethods>>(null);
+    useEffect(() => {
+      onNativeRender(() => {
+        // Not all components offer forwarded refs to native components. Only
+        // try to measure those who do.
+        if (!ref.current!.measure) {
+          pass();
+          return;
+        }
+
+        ref.current!.measure((_x, _y, width, height) => {
+          assert.isAtLeast(
```

Comment:
nit: Should we check X and Y are within the window? i.e `_x > 0` and `_x < (window - width)`

Reply:
There's an issue tracking it somewhere, but `ref.measure` currently doesn't return x or y coordinates correctly on RNW 😐